content/docs/nav.yml

```diff
@@ -1,88 +1,88 @@
-- title: Installation
+- title: Instalacja
   items:
     - id: getting-started
-      title: Getting Started
+      title: Podstawowe informacje
     - id: add-react-to-a-website
-      title: Add React to a Website
+      title: Dodaj Reacta do swojej strony
     - id: create-a-new-react-app
-      title: Create a New React App
+      title: Stwórz nową aplikację w Reakcie
     - id: cdn-links
-      title: CDN Links
-- title: Main Concepts
+      title: Linki do CDN-ów
+- title: Podstawowe informacje
   isOrdered: true
   items:
     - id: hello-world
-      title: Hello World
+      title: Witaj, świecie
     - id: introducing-jsx
-      title: Introducing JSX
+      title: Wprowadzenie do JSX
     - id: rendering-elements
-      title: Rendering Elements
+      title: Renderowanie elementów
     - id: components-and-props
-      title: Components and Props
+      title: Komponenty i właściwości
     - id: state-and-lifecycle
-      title: State and Lifecycle
+      title: Stan i cykl życia
     - id: handling-events
-      title: Handling Events
+      title: Obsługa zdarzeń
     - id: conditional-rendering
-      title: Conditional Rendering
+      title: Renderowanie warunkowe
     - id: lists-and-keys
-      title: Lists and Keys
+      title: Listy i klucze
     - id: forms
-      title: Forms
+      title: Formularze
     - id: lifting-state-up
-      title: Lifting State Up
+      title: Wynoszenie stanu w górę
     - id: composition-vs-inheritance
-      title: Composition vs Inheritance
+      title: Kompozycja a dziedziczenie
     - id: thinking-in-react
-      title: Thinking In React
-- title: Advanced Guides
+      title: Myślenie reactowe
+- title: Zaawansowane informacje
   items:
     - id: accessibility
-      title: Accessibility
+      title: Dostępność
     - id: code-splitting
-      title: Code-Splitting
+      title: Dzielenie kodu
     - id: context
       title: Context
     - id: error-boundaries
       title: Error Boundaries
```

Comment:
```suggestion
      title: Granice błędów
```

Reply:
WOW - nice - za chole.... nie wiedziałem jak to przetłumaczyć :-)